test-app/package.json

```diff
@@ -0,0 +1,137 @@
+    "broccoli-debug": "^0.6.3",
+    "broccoli-funnel": "^3.0.8",
+    "broccoli-merge-trees": "^4.2.0",
+    "chalk": "^4.1.1",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli-babel": "^8.0.0",
+    "ember-cli-build-config-editor": "^0.5.1",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-version-checker": "^5.1.2",
+    "ember-concurrency": "^2.1.2 || ^3.0.0",
+    "ember-element-helper": "^0.6.0 || ^0.7.0 || ^0.8.0",
+    "ember-focus-trap": "^1.0.0",
+    "ember-on-helper": "^0.1.0",
+    "ember-popper-modifier": "^2.0.0 || ^3.0.0 || ^4.0.0",
+    "ember-ref-bucket": "^4.0.0 || ^5.0.0",
+    "ember-render-helpers": "^0.2.0",
+    "ember-style-modifier": "^0.8.0 || ^1.0.0 || ^2.0.0 || ^3.0.0",
+    "findup-sync": "^5.0.0",
+    "fs-extra": "^11.0.0",
+    "resolve": "^1.18.1",
+    "silent-error": "^1.0.1",
+    "tracked-toolbox": "^1.2.3 || ^2.0.0"
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "7.23.3",
+    "@babel/plugin-proposal-decorators": "7.23.7",
+    "@ember/optional-features": "2.0.0",
+    "@ember/string": "3.1.1",
+    "@ember/test-helpers": "3.2.1",
+    "@embroider/test-setup": "2.1.1",
+    "@glimmer/component": "1.1.2",
+    "@glimmer/tracking": "1.1.2",
+    "@release-it-plugins/lerna-changelog": "6.0.0",
+    "bootstrap": "5.3.2",
+    "broccoli-asset-rev": "3.0.0",
+    "chai": "4.4.1",
+    "chai-things": "0.2.0",
+    "concurrently": "8.2.2",
+    "ember-a11y-testing": "6.1.1",
+    "ember-bootstrap": "workspace:*",
+    "ember-cli": "5.5.0",
+    "ember-cli-app-version": "6.0.1",
+    "ember-cli-blueprint-test-helpers": "0.19.2",
+    "ember-cli-content-security-policy": "2.0.3",
+    "ember-cli-dependency-checker": "3.3.2",
+    "ember-cli-fastboot": "4.1.2",
+    "ember-cli-fastboot-testing": "0.6.1",
+    "ember-cli-inject-live-reload": "2.1.0",
+    "ember-cli-sri": "2.1.1",
+    "ember-cli-terser": "4.0.2",
+    "ember-cli-test-loader": "3.1.0",
+    "ember-cli-version-checker": "^5.1.2",
+    "ember-compatibility-helpers": "1.2.7",
+    "ember-load-initializers": "2.1.2",
+    "ember-page-title": "8.2.0",
+    "ember-qunit": "7.0.0",
+    "ember-resolver": "11.0.1",
+    "ember-sinon": "5.0.0",
+    "ember-sinon-qunit": "7.4.0",
+    "ember-source": "5.5.0",
+    "ember-source-channel-url": "3.0.0",
+    "ember-template-lint": "5.13.0",
+    "ember-template-lint-plugin-prettier": "5.0.0",
+    "ember-try": "3.0.0",
+    "eslint": "8.56.0",
+    "eslint-config-prettier": "9.1.0",
+    "eslint-plugin-ember": "11.12.0",
+    "eslint-plugin-hbs": "1.0.3",
+    "eslint-plugin-n": "16.6.2",
+    "eslint-plugin-prettier": "5.1.3",
+    "eslint-plugin-qunit": "8.0.1",
+    "findup-sync": "^5.0.0",
+    "loader.js": "4.7.0",
+    "mocha": "10.2.0",
+    "prettier": "3.0.3",
+    "qunit": "2.20.0",
+    "qunit-dom": "3.0.0",
+    "release-it": "16.3.0",
+    "sinon": "17.0.1",
+    "tracked-toolbox": "^1.2.3 || ^2.0.0",
+    "webpack": "5.89.0"
+  },
+  "peerDependencies": {
```

Comment:
Can an app have peer dependencies?

Reply:
It can, but probably doesn't make any sense to. I'll move them to the addon package